src/main/scala/io/iohk/ethereum/faucet/FaucetConfig.scala

```diff
@@ -17,8 +24,7 @@ case class FaucetConfig(
     keyStoreDir: String,
     listenInterface: String,
     listenPort: Int,
-    minRequestInterval: FiniteDuration,
-    latestTimestampCacheSize: Int
+    minRequestInterval: FiniteDuration
```

Comment:
a few line above: ` corsAllowedOrigins: HttpOriginMatcher,` is this still needed? given we already have `rpc.http.cors-allowed-origins` ?

Reply:
same for port, and interface..

Reply:
Done